Lexicon.md

```diff
@@ -3,4 +3,5 @@
 * __Bit__: A boolean variable $$ b \in \{ 0, 1 \} $$. Represents the answer to a yes/no question.
 * __Bit String__: A sequence of __Bits__ of arbitrary length $$ B^n := (b_k)_{k=0}^n , \  k \in \mathbb{N} $$.
 * __Vulcan Set__: The set of all __Bit Strings__ of infinite length $$ \mathbb{V} := \{ B_0^{\infty}, B_1^{\infty}, \dots, B_{\infty}^{\infty} \} , \ B_i^{\infty} \neq B_j^{\infty} $$.
-* __Agent__: Any system that is best characterized as acting according to beliefs.
\ No newline at end of file
+* __Agent__: Any system that is best characterized as acting according to beliefs.
+* __Observable__: A unit of information that an __Agent__ has access to. This can be information acquired through any __Agent__ faculty. For example, human __Agents__ can __Observe__ sight, sound, thoughts, and many others. 
```

Comment:
By this definition if a human observes an object (e.g. the moon) is the moon the Observable, or the light from the moon, or the information in the brain of the observer?

Reply:
Test notifications

Reply:
I think the answer is both - The light is information is observed by the eyes, and the concept of the moon that is associated with the light pattern is then generated in the observer's brain and also observed.

light -> information that we interface with via our eyes
concept of moon -> information we interface with via our conscious/unconscious mind

Reply:
I'm trying to think of it as the concept of a moon is a reinterpretation of light information. Any accessible reinterpretation of information is still accessible information, and consequently still an observable.

Reply:
The moon, the moonlight, the qualia and the concept of the moon are four related but separate things. One observable or four?

Reply:
I'd say 3 observables - moonlight, qualia, concept. The moon itself is not directly an observable.

The same applies for pretty much anything, we can only talk about concept A insofar as we have interfaced with observables relating to concept A.

Reply:
Two of the things (qualia and concept) are in the mind and two are physical entities external to the agent (moon and moonlight). I'm confused how your definition can include 3 and exclude 1.

Reply:
perhaps "the moon" is an observable composed of the other three?

Reply:
The delineation between what is and is not an observable (that I had in mind) doesn't lie in whether the information is conceptual or physical. The crux is the information's accessibility. The reason I think the moon itself is not an observable, is because we cannot interface with it from earth with our bare senses directly. We can only interface with the moon through the light rays being reflected by it. We don't observe the moon itself, only its manifestation in terms of accessible information.

Maybe we can clarify problem further with the idea we posited on HipChat: that the conscious mind is a sub-agent of the human agent. We could consider two layers of agency in that case:

- Conscious mind: observes the concept and qualia of the moon.
- Physical human agent: observes the light rays from the moon (it then processes the information and passes it as a different set of observables to our conscious mind).

Reply:
Just posting what I was planning on talking about in the meeting I missed; I apologize since its a bit of a rant, I was thinking about it over the break. I'm going to start by summarizing my stance, and then address our points of contention.

### Abstract Observables

If we take a step back and consider a simple agent, A, the environment applies some observable O to A in a timestep cycle. This is the information that is contained in an observable. Say agent B pokes agent A. A would observe the feeling of being poked. If the poke is the only observation accessible by A, A would have to infer the existence of agent B, and the action that B took to elicit the feeling of being poked in A. A does not "observe" B, only the facilities with which B interfaces with A (i.e. the poke). This is similar to our issue of 'observing' the moon.

### Application to Human Agents, and Observation of the Moon

 I'm going to try to address the dynamics of how observables are applied to humans. This is not straightforward, because there is a discrepancy between the raw physical rendition of a human agent, and what we colloquially mean when we use the term "I". When I say "I observe", I am saying that my conscious inner agent is observing something, not that my physical agent is observing it. 

There is a layer of processing between observables accessed by our physical agents, and their renditions in our conscious agents. Back to our example, photons from our environment, including moon rays strike the frontal part of our face. They are all observables. I could encode the timing and location of the photons into a finite bit string. Those photons that happen to strike our eyes in the right way happen to transfer some more meaningful into us, but that is abstracted away by the implementation of our physical agent. These light rays transfer information into our rod and cone cells, which trigger electrical impulses that that are processed in a very complex way, and then the image of the sky with a grayish circle in it (the qualia of the moon) are presented to our conscious agent. We then associate that grayish circle with the concept we know as the moon. Like with agents A and B above, we never 'observe' the moon directly, only the way in which it pokes us. 

### Deviations from this Model 

Cecily brought up a good point, what if we ate an object? Would we be observing it directly then? I think the answer would depend on how we define our agent and encode the information of the observable. If we looked at an amoeba, where it swallows its food, and we could define the agent interface in 2 (or more) ways:

1. Anything physically outside the amoeba is the environment. When the amoeba swallows food, the food can be encoded into an observable and passed to the agent. In this case, the food _would_ be directly observable.
2. Anything physically outside the amoeba's cell membrane is the environment, including internal bubbles such as food vacuoles. In this case, the food would not be encoded as an observable, but the later digested food that enters the cell membrane would be.

### Summary

So to return to David's original question, in the case of the moonlight, the qualia, the concept of the moon, and the moon itself, I would still assert that only the first 3 are observable to us, but I don't think its a hard rule that objects themselves cannot be observables; it depends on the definition of the agent boundary, and how the observable information is encoded and passed into the agent.

### Potential Alterations

I don't think the definition of observable itself is under contention, but more the word "observable" itself. Maybe we could change it?
David mentioned that it would be difficult to have this lexicon be adopted if we cannot use colloquial phrases such as "The moon is observable". The one 'pro' that I would see with keeping the word "observable" for this concept is that it would fall in line with AIXI terminology. Though I agree that it can lead to confusion.

Reply:
At the weekly meeting @SeverTopan and I came to the tentative agreement that an "observation" is information that crosses an agent boundary from the outside and an "action" is information that crosses the boundary from the inside.

Reply:
I've updated this pull request to reflect this.